metadata/alpha.xml

```diff
@@ -7,7 +7,7 @@
 		<option name="modifier" type="key">
 			<_short>Modifier</_short>
 			<_long>When the modifier key is held down, you can scroll down and up to adjust the opacity of the window.</_long>
-			<default>&lt;alt&gt;</default>
+			<default>&lt;super&gt;</default>
```

Comment:
This should be retained so zoom can remain super+scroll.

Reply:
I rather prefer a single modifier.  <kbd>Super</kbd> being the default.

Reply:
I think zoom is generally more important than alpha, so we can use super+scroll for zoom, and here something else.

Reply:
Maybe put super+alt for alpha